toolchains/syno-purley-6.2.2/Makefile

```diff
@@ -16,7 +16,13 @@ TC_TARGET = x86_64-pc-linux-gnu
 TC_CFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
 TC_CPPFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
 TC_CXXFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
-TC_LIBDIR = $(TC_BASE_DIR)/sys-root/lib
-TC_LDFLAGS = -L$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_LIBDIR)
+TC_LDFLAGS = -L$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/lib
+
+FIX_TARGET = myFix
 
 include ../../mk/spksrc.tc.mk
+
+.PHONY: myFix
+myFix:
+	chmod -R u+w $(WORK_DIR)
+	@find $(WORK_DIR)/$(TC_BASE_DIR) -type f -name '*.la' -exec sed -i -e "s|^libdir=.*$$|libdir='$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/lib'|" {} \;
```

Comment:
This "myFix" target is copy-paste in many "recent" toolchains... I would move the target itself in `spksrc.tc-fix.mk` with a relevant "default target" (probably the recent version) the same way `spksrc.install.mk` provides `install_target` and `install_destdir_target` that can be used as `INSTALL_TARGET` value.